src/addbook.js

```diff
@@ -0,0 +1,15 @@
+import { todolist, input } from './delete.js';
+
+const addBook = () => {
+  const task = {
+    description: input.value,
+    completed: false,
+  };
+  todolist.push(task);
+  localStorage.setItem('list', JSON.stringify(todolist));
+  input.value = '';
+  window.location.reload();
+  return false;
```

Comment:
When a user adds a new task to do please just append the new task to the main container so that it can view it in real time when changes occur, we do not need to reload every time a user creates a new task.

Reply:
Can you give a detailed direction for this, I tried to fix this but I end up getting double the list in real-time.